loginradius-sdk/package.json

```diff
@@ -1,7 +1,10 @@
 {
   "name": "loginradius-sdk",
   "readme": "",
-  "scripts": {},
+  "scripts": {
+    "lint": "prettier --check sdk/**/*.js && eslint sdk/**/*.js",
+    "lint:fix": "prettier --write sdk/**/*.js  && eslint sdk/**/*.js --fix"
```

Comment:
how about change to ` "lint:fix": "npm run lint --fix"`?

Reply:
In order to pass flags when npm is invoked, we have to execute that as `npm run lint -- --fix`, however, it is quite usual to have separate scripts for linting and fixing linting issues.